public/js/main.js

```diff
@@ -0,0 +1,58 @@
+$(document).ready(function () {
+
+  //navigation bar-smooth scrolling by Paul Underwood
+  $('a[href^="#"]').on('click',function (e) {
+      e.preventDefault();
+
+      var target = this.hash;
+      var $target = $(target);
+
+      $('html, body').stop().animate({
```

Comment:
So, I know you're pulling this in from a blog post (and I really appreciate that you pointed it out), but for the sake of posterity -- the `stop` method here is unnecessary. Normally you'd include that to either pause an active animation or prevent one from firing. Here you don't actually need to worry about that.